{
  "contest": 189,
  "handle": "Rahoz",
  "risk": "2",
  "title": "Transfer can fail if receiver contain logic inside fallback",
  "issueId": 300,
  "issueUrl": "https://github.com/code-423n4/2022-12-escher-findings/issues/300"
}